src/libsystemd-network/fuzz-dhcp-client.c

```diff
@@ -16,7 +16,9 @@ int dhcp_network_bind_raw_socket(
                 uint32_t id,
                 const struct hw_addr_data *hw_addr,
                 const struct hw_addr_data *bcast_addr,
-                uint16_t arp_type, uint16_t port) {
+                uint16_t arp_type, uint16_t port,
```

Comment:
Not your fault, but please break the line before `uint16_t port`. That is, one argument per line.